### PR TITLE
LOK-2013: Re-enable dark mode

### PR DESCRIPTION
**Pull request by mikewrosey**
## Description
- Re-enables dark mode.
- Fixes dark mode on flows and dashboard graphs by removing hardcoded hex values and replacing them with feather categorical colors listed [here](https://feather.nanthealth.com/demos#/Styles-Status). 
- Fixes dark mode on the top bar by replacing the AppBar component with the AppHeader component.

Note: There is a bug in the feather App Rail for which an issue has been opened [here](https://github.com/feather-design-system/feather-design-system/issues/185)

Note2: In order to take advantage of the AppHeader and latest feather styles, I updated all feather packages. It's possible that this could cause unforeseen styling issues, but we don't have much of a choice and need to update at some point.

## Jira link(s)
- https://opennms.atlassian.net/browse/HS-2013
- https://opennms.atlassian.net/browse/HS-2014
- https://opennms.atlassian.net/browse/HS-2015
- https://opennms.atlassian.net/browse/HS-1705

The top bar fix for light mode:
screenshot-localhost_3009-2023 09 06-14_53_59

Flows line chart dark:
screenshot-localhost_3009-2023 09 06-14_53_32

Flows table chart dark:
screenshot-localhost_3009-2023 09 06-14_53_10

Dashboard polar chart test dark:
screenshot-localhost_3009-2023 09 06-14_00_00

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: ui/src/components/Dashboard/DashboardApplications.vue [hunk 1]
```diff
@@ -86,6 +92,15 @@ const config = {
       ticks: {
         display: false
       }
+    },
+    r: {
+      grid: {
+        color: isDark.value ? 'rgba(255, 255, 255, 0.1)' : 'rgba(0, 0, 0, 0.1)'
```
Comment (synqotik) [inline ui/src/components/Dashboard/DashboardApplications.vue:98]: This is used in several places, perhaps make a function in `utils.ts` like `getDefaultTextColor(isDark: boolean)` or similar.

**Review by synqotik — APPROVED** [review]
Approved, with one suggestion.

**Review by freddy61025 — APPROVED** [review]
LGTM & agree on Scott. It won't block merge but nice to have.

File: ui/src/components/Dashboard/DashboardApplications.vue (hunk 1 above)
Comment (mikewrosey) [reply ui/src/components/Dashboard/DashboardApplications.vue]: Good suggestion, done!

**Pull request merged**